utils/build-llvm-cross-local.sh

```diff
@@ -78,19 +77,19 @@ cmake ../llvm \
   -DMLIR_INCLUDE_INTEGRATION_TESTS=OFF \
   -DLINKER_SUPPORTS_COLOR_DIAGNOSTICS=OFF \
   -DLLVM_ENABLE_TERMINFO=OFF \
-  -DLLVM_DEFAULT_TARGET_TRIPLE="aarch64-linux-gnu" \
-  -DLLVM_TARGET_ARCH="AArch64" \
-  -DCMAKE_CXX_STANDARD=17 \
```

Comment:
This is already specified in [`llvm/CMakeLists.txt`](https://github.com/llvm/llvm-project/blob/3fe69bab1149e20f87ac31f8b2557baa88fcdd14/llvm/CMakeLists.txt#L70).